src/pudl/analysis/record_linkage/embed_dataframe.py

```diff
@@ -108,6 +108,42 @@ def embed_dataframe_graph(df: pd.DataFrame) -> FeatureMatrix:
     return embed_dataframe_graph
 
 
+def dataframe_cleaner_factory(
```

Comment:
@zschira I made this "cleaner factory" which is very similar to the "embedding factory" except that it doesn't have `weights` and also the output of the transform and fit steps is a pandas dataframe instead of a numpy array or spare matrix. It might be cleaner to just adapt the embedding factory to work for both? Not sure. They serve different purposes - the embedding factory results in a matrix embedding versus the cleaner factory is only intended to clean up some columns. I could see how this can be confusing to a user.

Reply:
I'll make an issue in SEC-EIA land to refactor this.